src/common.tests/TestData/simpleconfig.yml

```diff
@@ -0,0 +1,26 @@
+    server: https://sample-demo.hcp.eastus.azmk8s.io:443
+  name: sample-aks
+contexts:
+- context:
+    cluster: sample-aks
+    namespace: todo-app-ingress
+    user: clusterUser_sample-demo_sample-aks
+  name: sample-aks
+current-context: sample-aks
+kind: Config
+preferences: {}
+users:
+- name: clusterUser_sample-demo2_sample-aks
+  user:
+    client-certificate-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUZIVENDQXdXZ0F3SUJBZ0lRRU10SGwxeVk0YldiVTF2SHJXVXdrREFOQmdrcWhraUc5dzBCQVFzRkFEQU4KTVFzd0NRWURWUVFERXdKallUQWVGdzB5TXpBeU1UTXdPVEF3TWpsYUZ3MHlOVEF5TVRNd09URXdNamxhTURBeApGekFWQmdOVkJBb1REbk41YzNSbGJUcHRZWE4wWlhKek1SVXdFd1lEVlFRREV3eHRZWE4wWlhKamJHbGxiblF3CmdnSWlNQTBHQ1NxR1NJYjNEUUVCQVFVQUE0SUNEd0F3Z2dJS0FvSUNBUURDYzMrWXVYUGNlZm50UEpoOWdNY3AKTmtlclVQRFA0SUk5R3U4aVNFbmlDK1FFUWdZUklKTFJTUkphZXRicU9JTGIwMTFSaFlDSzd3Qms2bjlNNEJiTApUTUlOekcwVG5GWmJFbDRMWjNtOW5tME96Mld4QUFoNFFYTGxjcWlIQ1hiK1J0ZUlRdVA1SG85Tk1mR2FNVFVRCmhSQnZOcEMzbXNONUFPVzRwUG5Xa25kOXVONS9qZWtaaW9iOVpEUENidGJ4SjdsZnJ1OHZiNU9pZFhCZzlFSUwKQWNHanNyd1F3K1MzaXZ5Y0JycnlFOUsyU1dDTnZyNUNlVXRPdjArTTA4cHBzUkRyWjIyR0RkSWxIQ2pwU0xQUwpRdTAzaGpxNnE0TUNlOTJIVy9aV3RVK2hhQzNwKzhuOXJRd2NxaXpiZmptWGNVNVM0RUJFZitVYU1hYms3cGxoCm5kR1Iwd2lMSDRnZDNFOWFvUHI1MFhrTVFFdkRrZW94NTRTNlo4Y1pmblhodkJ3RG5haHM4RiszQ3FRWlAxaFkKUC8vSGNLcEsxSzZjb1RuQ3pDQ2M0TXZIdFhGK1dWVHFVdndsNGN6MFYvaDlvS0VGMVY3U1Y3L2RKVU52Y05TNQo0OWFyTlpKUWMzRW12VENzUDdmOG8vTjJHTlVVaW92RmRrQ3hmdk1mWGhtbE16YjAvSXFkaWVTNkdhTVAzQ0YwCk4xU1hPUzFxcncrSFBaYVpLN2c0ZjZNeTBUcjJ5dENNMld6VjJ6ZDUxdjUrWGcvWEpoUkhjWnJGOU1KRUVHVE4KeC9IUlkzSXZTMlFRZEtnMlk3WG9uU2hzaU4wRjhia0U3Uk9GanVMYWM4WGhXV2t2UGtVMnJOcUMyTGt2RmFKVQpGNmFkdzVOU1MvSFJzUVNkUWZUa2x3SURBUUFCbzFZd1ZEQU9CZ05WSFE4QkFmOEVCQU1DQmFBd0V3WURWUjBsCkJBd3dDZ1lJS3dZQkJRVUhBd0l3REFZRFZSMFRBUUgvQkFJd0FEQWZCZ05WSFNNRUdEQVdnQlNpYlpwOFZ6REoKL1EwL1NOZHNka1h5T2pSemFUQU5CZ2txaGtpRzl3MEJBUXNGQUFPQ0FnRUFVTEplOXBMd2pHMDJNUlBCRCtDQgp2ZVJoQWVYNHAzRUZNZEEvTGpyaEJvQk1jZkZGWitKNmtaeENnUjNDS2trZkI4c3gzL1pGU25TQy9xYVhaSGJQCmJlZUhtdWZNcHZQdVBjakdtaGpoQ2gwQzhPQ2JhRk05R2NEeTNlblk2a25aRUxab1prVkkzcy83bEsvdnJjZmMKYXd5WGlGb2JHc3N5d2swUk5XVjdmczA2WkV0RzA3c09jTXVZaStER2RzSmp0NmY3aVNDcENLbko1WENyQzhMRgo1eTNZWVlOcUM0YUgwR2ZZV2VrSHZyUUVQRGlQbjJITndCQmlVRTl2NlYxb3AzbmhEUTBRcTJrQ05aeFRFZU5lClloWVF1WThyYXg2RitrbVlFdGdvdzh6bS9vWGZ0UDhkc1JEY2lNZHhxQmZ1ZnRVNjBBd0xpeTVTcDZVMExGL0gKWlhaN1JGSlROeEU1cjhyOXYyblBUUnJLR1ZkQzhldjhQM3NIRHY3RkFNNjFpSE1aYnJ5bDFrOTk1Q0RhU0FNYwpyMU5tb21tSFpGR2lkSlROTUhTdHQvRDRvMVhmNUFvTHJPTXhjZCtaQy8rbWxaS3crQ01XTnJlaDFYWVIrVHR6Cm9TbmJKRE5WcGZrdDBrZlo5UVJHQTZ3dGlZNFVpYit1U1NiVGNtMEVId1h5T1d0S250em9pL2YrbCtsYjZ6V2IKVUxRa1J0d2xQMmRRUERnVUVJRm9QV3AyS1U2RlVWVExBOUt1WWRCRCtIV1YrbFpJeSs1a09sN2NTYjRSd00wRQpiSHliTzdEQm96M3NqWkxHbjd6dm1rU3ZLckp5eEZLaXVGZUxUNmdzRnROdHo1NXFveXNWNHBYWXh4bGZTRGJsCllmMWNKckNNMjgxdHJBRVVaaG1jZ1MwPQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```

Comment:
💡 Thought and idea please -> Hope this cert is not some real work data and totally random stuff, in that case just fuzzy this more.

Reply:
@Tatsinnit good catch, I was expecting our builds to fail but it didn't, builds do have creds scan not sure why it didn't fail, I will check further. Yes, these certs are not real (expired one though). I will remove them after fixing the build pipelines since I feel like still exposing these are not fair enough. Thoughts ?

Reply:
Added CredScan to PR builds. :)

Reply:
seems like cred scans ignores files in test folder. :).